src/main.py

```diff
@@ -49,37 +75,18 @@ async def start_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
         "We're here to make your experience at UCLA as enjoyable as possible! 😊\n\n" 
     )
 
-    # Create a menu with options
-    keyboard = [
-        [InlineKeyboardButton("Upcoming Events", callback_data="events")],
-        [InlineKeyboardButton("SSA Fams Leaderboard", callback_data="fam_points")],
-    ]
-    reply_markup = InlineKeyboardMarkup(keyboard)
+    # Call the function to create the menu
+    reply_markup = create_menu()
 
     await update.message.reply_text(welcome_message, reply_markup=reply_markup)
 
 async def help_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    await update.message.reply_text("What do you want")
-
-# Responses
-# AI will go in here later on to process inputs
-
-def handle_response(text: str) -> str:
```

Comment:
looks like you refactored this into the handle_message function? can you refactor it back out cos we'll eventually implement some kind of response handler (either chat AI or some fixed set of responses) but yea, it'll be better for future development

Reply:
it is currently still outside the handle_message function, idk why github shows the additions and deletions in a weird way but this is currently what it is for the help command

```
async def help_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
    # Call the function to create the menu
    reply_markup = create_menu()

    await update.message.reply_text("Here are our available menus:", reply_markup=reply_markup)
```